src/Containers/VendorProductCard/VendorProductCard.scss

```diff
@@ -22,6 +22,7 @@
   align-items: center;
   justify-content: center;
   border-right: 2px solid $dark_green_border_color;
+  color: black;
```

Comment:
This looks a little off too - the text for the description is unreadable since it is the same color as the background